src/proc/align.cpp

```diff
@@ -0,0 +1,205 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#include "../include/librealsense2/rs.hpp"
```

Comment:
Don't include `rs.hpp` nor `rsutil.h`

Reply:
I'll adjust it to the minimum set of includes required